File: clients/java3/src/org/openamq/client/framing/AMQFrameDecodingException.java
package org.openamq.client.framing;

import org.openamq.client.AMQException;

/**
 * @author Robert Greig (robert.j.greig@jpmorgan.com)
 */
public class AMQFrameDecodingException extends AMQException
{
    public AMQFrameDecodingException(String message)
    {
        super(message);
    }

    public AMQFrameDecodingException(String message, Throwable t)
    {
        super(message, t);
    }
}

File: dotnet/xmscommon/framing/AMQFrameDecodingException.cs
using System;
using log4net;

namespace OpenAMQ.Framing
{
    /// <summary>
    /// Thrown when a frame cannot be decoded. This generally indicates a mismatch between the broker and the
    /// client.
    /// </summary>
    public class AMQFrameDecodingException : AMQException
    {
        public AMQFrameDecodingException(string message)
            : base(message)
        {
        }

        public AMQFrameDecodingException(string message, Exception innerException)
            : base(message, innerException)
        {
        }

        public AMQFrameDecodingException(ILog logger, string message)
            : base(logger, message)
        {
        }

        public AMQFrameDecodingException(ILog logger, string message, Exception innerException)
            : base(logger, message, innerException)
        {
        }            
    }
}
